src/components/menu/menu.scss

```diff
@@ -38,6 +36,12 @@ ion-menu.show-menu {
   transform: translate3d(-9999px, 0, 0);
 
   contain: strict;
+
+  // scss-lint:disable PropertySpelling
+  @include multi-dir() {
+    left: 0;
+    right: auto;
+  }
```

Comment:
@AmitMY but without this left menu open at a wrong position especially in tablets... mind check it?